submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/query/SubmissionListApiTest.kt

```diff
@@ -151,58 +151,82 @@ internal class SubmissionListApiTest(private val tempFolder: TemporaryFolder) :
         }
 
         @Test
-        fun `submission with section title`() {
+        fun `get submissions with submission title`() {
             val submission = tsv {
                 line("Submission", "SECT-123")
+                line("Title", "Submission subTitle")
                 line()
 
                 line("Study")
-                line("Title", "Submission With Section Title")
+                line("Title", "Submission With Section")
                 line()
             }.toString()
 
             assertThat(webClient.submitSingle(submission, TSV)).isSuccessful()
 
             val submissionList = webClient.getSubmissions(
                 mapOf(
-                    "accNo" to "SECT-123"
+                    "keywords" to "subTitle"
```

Comment:
I think we should include spaces, i.e. "Section Title". It seems like a case the users might need

Reply:
https://github.com/EBIBioStudies/biostudies-backend-services/pull/395/files#diff-57ceb60d5cf13b0bf3e3bd1a58158ba778820910103525101b7cf15ff92630f0R206